cpp/src/Ice/PropertyNames.h

```diff
@@ -55,8 +56,7 @@ namespace IceInternal
         static const PropertyArray IceBTProps;
         static const PropertyArray Glacier2Props;
 
-        static const PropertyArray validProps[];
-        static const char* clPropNames[];
+        static const std::array<PropertyArray,14> validProps;
```

Comment:
Updated to used `std::array` now that we know the size up front.

Reply:
need a space after the comma.

Reply:
Can you remove the ICE_API and make the .cs class internal?